.github/workflows/publish-commit-bottles.yml

```diff
@@ -40,6 +40,13 @@ jobs:
           HOMEBREW_BINTRAY_KEY: ${{secrets.HOMEBREW_BINTRAY_KEY}}
         run: brew pr-pull --debug ${{github.event.inputs.args}} ${{github.event.inputs.pull_request}}
 
+      - name: Pull bottles (wheezy)
```

Comment:
This could be parametrized by workflow inputs. Create a new `workflow` input on top with `default: tests.yml` and set `--workflow` flag for `brew pr-pull` then.

Reply:
Oh, disregard the above comment.

Reply:
We need to run pr-pull twice, to fetch the Mac and the linux bottles